src/utils/permission/index.js

```diff
@@ -86,6 +86,7 @@ export const requestExternalStoragePermission = async () => {
     if (
       requestResult['android.permission.READ_EXTERNAL_STORAGE'] === 'granted' ||
       requestResult['ios.permission.PHOTO_LIBRARY'] === 'granted' ||
+      requestResult['ios.permission.PHOTO_LIBRARY'] === 'limited' ||
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the check for 'limited' access to the photo library is a good move considering iOS 14+ allows users to select specific photos to grant access to, instead of the entire photo library. However, it's important to note that with 'limited' access, not all features might work as expected if they require access to the entire photo library. You should ensure that your application can handle this case gracefully.
```diff
+ requestResult['ios.permission.PHOTO_LIBRARY'] === 'limited' ||
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->